wp-content/plugins/core/src/Blocks/Types/Button.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Blocks\Types;
+
+use Tribe\Gutenpanels\Blocks\Block_Type_Interface;
+use Tribe\Gutenpanels\Blocks\Sections\Content_Section;
+use Tribe\Project\Blocks\Block_Type_Config;
+
+class Button extends Block_Type_Config {
+
+	public const NAME = 'tribe/button';
+
+	public const LINK = 'link';
+	public const ARIA = 'aria_label';
+
+	public function build(): Block_Type_Interface {
+		return $this->factory->block( self::NAME )
+			->set_label( __( 'Button', 'tribe' ) )
+			->set_dashicon( 'align-center' )
```

Comment:
Will we need a custom icon for our button block?

Reply:
Yes, we will need a custom icon. I'm waiting on Sam to set up a framework for that.